projects/core/pipes/flag/flag.pipe.ts

```diff
@@ -0,0 +1,32 @@
+import {Inject, Pipe, PipeTransform} from '@angular/core';
+import {TUI_SVG_OPTIONS, TuiSvgOptions} from '@taiga-ui/core/components/svg';
+import {TuiCountryIsoCode} from '@taiga-ui/i18n';
+
+@Pipe({name: `tuiFlag`})
+export class TuiFlagPipe implements PipeTransform {
+    private readonly staticPath = this.svgOptions
+        .path(`tuiIcon`)
+        .replace(`tuiIcon.svg#tuiIcon`, ``);
+
+    constructor(@Inject(TUI_SVG_OPTIONS) private readonly svgOptions: TuiSvgOptions) {}
+
+    transform(countryIsoCode: TuiCountryIsoCode): string {
+        switch (countryIsoCode) {
```

Comment:
it would be great to be able to specify also custom handler

```ts
transform(countryIsoCode: TuiCountryIsoCode | string): string {
  return this.customHandler?.(countryIsoCode) ?? this.defaultHandler(countryIsoCode);
}

defaultHandler(countryIsoCode: TuiCountryIsoCode): string {
   switch (countryIsoCode) {
            case TuiCountryIsoCode.BL:
            case TuiCountryIsoCode.BQ:
            case TuiCountryIsoCode.CW:
            case TuiCountryIsoCode.GF:
            case TuiCountryIsoCode.GP:
            case TuiCountryIsoCode.MF:
            case TuiCountryIsoCode.MQ:
            case TuiCountryIsoCode.NC:
            case TuiCountryIsoCode.RE:
            case TuiCountryIsoCode.YT:
                return `${this.staticPath}${TuiCountryIsoCode.FR}.png`;
            case TuiCountryIsoCode.SX:
                return `${this.staticPath}${TuiCountryIsoCode.NL}.png`;
            default:
                return `${this.staticPath}${countryIsoCode}.png`;
        }
    }
}

```
```

Reply:
I think developer should create its own pipe for this case 🤔

Reply:
@waterplea @vladimirpotekhin 
guys, what do you think ?
Should we add custom handler as the second argument of the pipe (or DI token) ?

Reply:
`I think developer should create his/her own pipe for this case` Yes, you are right